code/game/gamemodes/endgame/scoreboard/department_score.dm

```diff
@@ -0,0 +1,176 @@
+					disease_spread_count++
+					if (L.stat != DEAD)
+						score.disease_good++
+		else
+			for (var/mob/living/L in mob_list)
+				if(!L.mind) //No ballooning the negative score with infected monkeymen
+					continue
+				if (ID in L.virus2)
+					disease_spread_count++
+					if (L.stat != DEAD)
+						score.disease_bad++
+
+		if (disease_spread_count > score.disease_most_count)
+			score.disease_most_count = disease_spread_count
+			score.disease_most = ID
+	/*for(var/mob/living/carbon/human/I in mob_list)
+
+		if(I.viruses) //Do this guy have any viruses ?
+			for(var/datum/disease/D in I.viruses) //Alright, start looping through those viruses
+				score.scores["disease"]++ //One point for every disease
+	*/
+	//var/beneficialpoints = score.scores["disease_good"] * 20
+	score.disease_vaccine = ""
+	for (var/antigen in all_antigens)
+		if (isolated_antibodies[antigen] == 1)
+			score.disease_vaccine += "[antigen]"
+			if (antigen in blood_antigens)
+				score.disease_vaccine_score += 40
+			else if (antigen in common_antigens)
+				score.disease_vaccine_score += 120
+			else if (antigen in rare_antigens)
+				score.disease_vaccine_score += 200
+			else if (antigen in alien_antigens)
+				score.disease_vaccine_score += 400
+		else
+			score.disease_vaccine += "-"
+
+	if (score.disease_vaccine_score == 2320)
+		score.disease_vaccine_score = 3000 // panacea bonus
+
+	//crewscore
+	score.crewscore -= score.disease_bad * 50 //A diseased crewman is half-dead, as they say, and a double diseased is double half-dead
+	score.crewscore += score.disease_vaccine_score
+	score.crewscore += score.disease_effects
+
+	return dat
+
+/datum/controller/gameticker/scoreboard/proc/engineering_score(var/dat = "")
+	var/skip_power_loss = 0
+	for(var/datum/event/grid_check/check in events)
+		if(check.activeFor > check.startWhen && check.activeFor < check.endWhen)
+			skip_power_loss = 1
+	if(!skip_power_loss)
+		for(var/obj/machinery/power/apc/A in power_machines)
+			if(A.z != map.zMainStation)
+				continue
+			for(var/obj/item/weapon/cell/C in A.contents)
+				if(C.percent() < 30)
+					score.powerloss++ //Enough to auto-cut equipment, so alarm
+	for(var/datum/powernet/PN in powernets)
+		if(PN.avail > score.maxpower)
+			score.maxpower = PN.avail
+	//var/atmos
+	//if(score.scores["airloss"] != 0)
+		//atmos = score.scores["airloss"] * 20 //Air issues are bad, but since it's space, don't stress it too much
+	//if(!atmos) //No air alarms anywhere
+		//score.crewscore += 5000 //Give the Atmospheric Technicians a good pat on the back for caring
+		//score.scores["atmosbonus"] = 1
+
+	if(!score.powerloss) //No APCs with bad power
+		score.powerbonus = 2500
+
+	//crewscore
+	score.crewscore -= score.powerloss * 50 //Power issues are BAD, they mean the Engineers aren't doing their job at all
+	score.crewscore += score.powerbonus
+	//score.crewscore -= atmos
+	return dat
+
+/datum/controller/gameticker/scoreboard/proc/service_score(var/dat = "")
```

Comment:
These procs are defined with a `dat` parameter, which they return, but don't use. They are also called without this argument. Was this an oversight?

Reply:
I just wanted to format the procs so that they _could_ use the dat parameter if in the future it's used. Just some future-proofing.

Reply:
I don't think that makes sense

Reply:
modified it so the procs pass the dat